spec/amaysim_logger/rails_controller_helper_spec.rb

```diff
@@ -33,7 +33,7 @@ class TestController < ActionController::Base
         # rubocop:disable RSpec/ExampleLength
         it 'logs the http request with a generated correlation id' do
           log = {
-            msg: 'controller request',
+            msg: 'API request',
```

Comment:
thanks changed this to web request.  Needs to be sumo logic business report friendly.